BackendApartmentReservation/BackendApartmentReservation/Managers/EmployeeManager.cs

```diff
@@ -14,10 +14,11 @@ public EmployeeManager(IEmployeeRepository employeeRepository)
             _employeeRepository = employeeRepository;
         }
 
-        public async Task<string> CreateEmployee(string firstName, string lastName, string email, string office)
+        public async Task<string> CreateEmployee(DbEmployee dbEmployee)
         {
-            var employeeId = Guid.NewGuid().ToString();
-            await _employeeRepository.CreateEmployee(firstName, lastName, email, office);
+            var employeeId = Guid.NewGuid().ToString() ;

```

Comment:
This ID must be used in the repository and in entity, probably as `ExternalEmployeeId`